src/main/webapp/components/BaseFormView.jsx

```diff
@@ -485,6 +485,9 @@ class BaseFormView extends PureComponent {
                 this.isOAuth &&
                 (this.isSingleOauth || (this.isAuthVal && this.datadict.auth_type === 'oauth'))
             ) {
+                // clear out Basic Authentication fields before doing OAuth Authentication
+                this.datadict.username = '';
+                this.datadict.password = '';
```

Comment:
This is static. We need to use `globalConfig.json` to get field names dynamically and use that. There should be a variable already defined as far as I know which tracks fields for oauth.

Reply:
For reference:
https://github.com/splunk/addonfactory-ucc-base-ui/blob/c222d1a7bb0b13730f8e21fef051cc5cc1a48ad3/src/main/webapp/components/BaseFormView.jsx#L455-L459

Reply:
Done